### DISCUSSION TITLE
Can't activate license key, no response from pyarmor@163.com

### DISCUSSION THREAD

**Discussion by subatiq** [dashingsoft/pyarmor#1248] [Q&A] [answered]
Hello, I get 
> this code has been used too many times, please send order information to pyarmor@163.com to activate it. If this request is not sent from register email, no reply and nothing to do. More information refer to https://github.com/dashingsoft/pyarmor/issues/980

when trying to register 7.7.0 key using second method described in the registration file "pyarmor-regcode-xxxxxx.txt".
I assume our CI autobuilds accidentally used all 10 attempts.

I've sent the info to pyarmor@163.com but did not get any response in 12 days. Would be grateful for any help on that.

**Comment by jondy** [comment]
What's your order no.? I remember I have handled all of this kind of issue? First make sure your email server doesn't block pyarmor@163.com, also check spam folder

**Comment by subatiq** [comment]
Order number 797339893
Strange, maybe something is blocking mail, but I can't figure out what

**Comment by jondy** [comment, marked as answer]
@subatiq please send order information to pyarmor@163.com from registration email, for this kind of this issue, only registration email works.

**Reply by subatiq** [reply]
Got it, just sent another request from registration email
